pkg/storage/etcd/testing/testingcert/certificates.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package testing
+package testingcert
```

Comment:
At some point in 1.5 we should rationalize the location of some of these testing bits that are shared across implementations.